singa_easy/modules/explanations/maskmap/misc_functions.py

```diff
@@ -203,7 +209,7 @@ def recreate_image(im_as_var):
         recreated_im (numpy arr): Recreated image in array
     """
     reverse_mean = [-0.485, -0.456, -0.406]
-    reverse_std = [1/0.229, 1/0.224, 1/0.225]
+    reverse_std = [1 / 0.229, 1 / 0.224, 1 / 0.225]
```

Comment:
is this required by the pylint? or done by the autoformat tool?
1/0.229 is faster to code:-)

Reply:
this is done by the autoformat tool, it adds a space